ui/src/lib/gql/project/useApi.ts

```diff
@@ -26,7 +34,7 @@ export const useProject = () => {
     }
   };
 
-  const useGetProjects = (workspaceId: string): GetProjects => {
+  const useGetWorkspaceProjects = (workspaceId: string): GetWorkspaceProjects => {
```

Comment:
Wasn't useGetProjects meaningful enough? Projects will only ever be workspace projects

Reply:
So I added the `getProject` which can be used to get a single project which got me thinking. 
This is more explicit stating that `projects` belong to `workspace` hence the name.

Reply:
And workspace ID needs to be supplied. There's also a query (not implemented yet) which is `GetProjects` where you can pass in the `IDs` and set `Node` to `Project` and get all the projects with that `IDs` it was much clearer in my head to rename it as `workspaceProjects`